vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -541,8 +541,9 @@ func (e *StandardTxExecutor) ConvertSubnetTx(tx *txs.ConvertSubnetTx) error {
 	avax.Consume(e.State, tx.Ins)
 	// Produce the UTXOS
 	avax.Produce(e.State, txID, tx.Outs)
-	// Set the new Subnet manager in the database
-	e.State.SetSubnetManager(tx.Subnet, tx.ChainID, tx.Address)
+	// Track the subnet conversion in the database
+	// TODO: Populate the conversionID
```

Comment:
Any reason to not calculate the conversion ID here now? Seems somewhat trivial since all the data to calculate is available.

Reply:
The `SubnetConversionTx` changes are in #3397. So those are already pulled out into a separate PR.